index.js

```diff
@@ -55,6 +57,39 @@ const createResponse = (responseMap, text) => {
   return response;
 }
 
+const createWorkflowResponse = (workflows) => {
+  const response = Object.assign({}, workflowTemplate);
+
+  const items = workflows.map(workflow => {
+    return {
+        type: "action",
+        action: {
+            type: "workflow",
+            workflowId: workflow._id,
+            label: workflow.title,
+        }
+    }
+  });
+
+  response.quickReply.items = items;
+
+  if (items.length === 0) {
+    response.text = "We couldn't find any workflow that matched the search criteria";
+  }
+
+  return [response];
+}
+
+const searchWorkflow = async (client, replyToken, userId, keyword) => {
```

Comment:
Let's have this function just search the workflow as the name suggests. So it should return the workflows.

Reply:
Done! You can have a look at the new commit.